src/notifications/Manager.java

```diff
@@ -0,0 +1,18 @@
+package notifications;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager implements Observer {

```

Comment:
This object could be an EmailNotificator or EmailService that keeps a mailing list with Market Manager's email.